### ISSUE TITLE
Template generation fails with sbt 1.4.0

### ISSUE THREAD

**Issue by PashoQ** [foundweekends/giter8#476]
## steps

sbt 1.4.0, Giter8 version: 0.13.1

## problem

sbt new fails to generate project

<details>

<summary>when generating template sbt request http auth. But after typing any character for Username and/or Password it fails with error bellow</summary>

```scala
PS H:\t> sbt new https://github.com/PashoQ/oozie.g8.git
[info] welcome to sbt 1.4.0 (AdoptOpenJDK Java 1.8.0_265)
[info] loading settings for project global-plugins from dependency-graph.sbt,updates.sbt ...
[info] loading global plugins from C:\Users\id967052\.sbt\1.0\plugins
[info] set current project to t (in build file:/H:/t/)
[info] set current project to t (in build file:/H:/t/)
Username: Password:
java.io.IOException: The parameter is incorrect
```

</details>

## expectation

It should generate tempate

**Comment by PashoQ** [comment]
Was fixed by https://github.com/sbt/sbt/issues/5952. What is remaining is echo when entering parameters values.

**Comment by PashoQ** [comment]
Fixed with SBT 1.4.2 release
